subprojects/platform-play/src/main/java/org/gradle/play/internal/platform/PlayMajorVersion.java

```diff
@@ -60,6 +61,9 @@ public static PlayMajorVersion forPlatform(PlayPlatform targetPlatform) {
     public static PlayMajorVersion forPlayVersion(String playVersion) {
         VersionNumber versionNumber = VersionNumber.parse(playVersion);
         if (versionNumber.getMajor() == 2) {
+            if (versionNumber.getMinor() == 2) {
+                DeprecationLogger.nagUserWith("Play 2.2 support was deprecated and will be removed in Gradle 5.0. Please upgrade your Play version.");
```

Comment:
We may want to introduce a new method in `DeprecationLogger` that reuses `getDeprecationMessage()` so you don't have to hard code Gradle 5.0.

Reply:
Done.